src/components/CountriesList.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FaRegArrowAltCircleRight } from 'react-icons/fa';
+import { getCountries } from '../redux/countriesSlice';
+
+const CountriesList = () => {
+  const dispatch = useDispatch();
+  const [searchQuery, setSearchQuery] = useState('');
+  const { countriesList, isLoading } = useSelector((store) => store.countries);
+
+  useEffect(() => {
+    dispatch(getCountries());
+  }, [dispatch]);
+
+  const handleChange = (e) => {
+    setSearchQuery(e.target.value);
+  };
+
+  const filteredCountries = countriesList.filter(({ name }) => name.common.includes(searchQuery));
+
+  if (isLoading) {
+    return <h2>Loading data...Please wait!</h2>;
+  }
+
+  return (
+    <div className="container">
+      <h1>WORLD COUNTRIES INFO</h1>
+      <form>
+        <input
+          type="search"
+          placeholder="Search by country name..."
+          value={searchQuery}
+          onChange={handleChange}
+        />
+      </form>
+      {filteredCountries.map((country) => (
+        <div key={country.flag} className="country">
+          <Link to={`details/${country.flag}`}>
+            <FaRegArrowAltCircleRight />
+          </Link>
+          <img src={country.flags.png} alt={country.flags.alt} />
+          <p>{country.name.common}</p>
+          <p>
+            Population :
+            {country.population}
+          </p>
+        </div>
+      ))}
+    </div>
```

Comment:
- Good job done so far, but your homepage is not styled according to the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏 According to the project requirements, you are required to follow the below design guidlines:
-- Typography: font face, size, and weight
-- Layout: composition and space between elements. Please fix the composition and space between your elements 🙏 

Your project:
![awtaw4tw](https://user-images.githubusercontent.com/73607512/232822147-69694bf9-0ed8-4df5-80e3-e2b609001808.PNG)